src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -670,12 +674,50 @@ def test_graph_group_scenario(self):
             self.cmd('ad group member check -g {child_group_id} --member-id {leaf_group_id}',
                      checks=self.check('value', False))
 
+            # create user to add group member
+            user_result = self.cmd('ad user create --display-name {user1} --password {pass} --user-principal-name {user1}@{domain}').get_output_in_json()
+            self.kwargs['user1_id'] = user_result['id']
+
+            # add user as group member
+            self.cmd('ad group member add -g {leaf_group_id} --member-id {user1_id}')
+
+            # check user as group member
+            self.cmd('ad group member check -g {leaf_group_id} --member-id {user1_id}',
+                     checks=self.check('value', True))
+            # list member(user is expected)
+            self.cmd('ad group member list -g {leaf_group_id}', checks=self.check('length([])', 1))
+
+            # remove user as member
+            self.cmd('ad group member remove -g {leaf_group_id} --member-id {user1_id}')
+            self.cmd('ad group member check -g {leaf_group_id} --member-id {user1_id}',
+                     checks=self.check('value', False))
+
+            # Create service principal to add group member
+            app = self.cmd('ad app create --display-name {app_name} --identifier-uris api://{app_name}').get_output_in_json()
+            self.kwargs['app_id'] = app['appId']
+            sp = self.cmd('ad sp create --id {app_id}').get_output_in_json()
```

Comment:
Creating `sp` immediately after creating an `app` can lead to failure. Let's use `az ad sp create-for-rbac` which has retry logic for `sp` creation. 

See https://github.com/Azure/azure-cli/issues/21674